.github/workflows/boot-api.image+upload.yaml

```diff
@@ -71,3 +71,6 @@ jobs:
         run: |
           docker tag orakl-boot-api ${{ env.ecr_url }}:v0.0.1.${{ needs.prepare.outputs.tag_date }}.${{ needs.prepare.outputs.tag_git_hash }}
```

Comment:
Is it on purpose to have the version fixed (`v0.0.1`) Same for `node` below.

Reply:
yes, version can be told through date and git hash